mantle/cmd/kola/testiso.go

```diff
@@ -89,7 +89,8 @@ var (
 		"iso-offline-install.bios",
 		"iso-offline-install.mpath.bios",
 		"iso-offline-install-fromram.4k.uefi",
-		"iso-offline-install-iscsi.bios",
+		"iso-offline-install-iscsi.ibft.bios",
```

Comment:
Renaming is OK, but make sure that there is no mention of it in any of the existing denylist files.

Reply:
https://github.com/openshift/os/pull/1461